modules/installation-nutanix-agent-installer-config-yaml.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+
+// * installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-nutanix-agent-installer-config-yaml_{context}"]
+= Sample install-config.yaml file for Nutanix
+
+You can customize the `install-config.yaml`` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+* Installing Nutanix virtual machines (VMs) on {aws-first} is not supported.
+
+* Before you begin an installation, you must change the boot order of all Nutanix VMs to prioritize the disk first.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: test.metalkube.org <1>
+controlPlane: <2>
+  name: master
+  replicas: 3 <3>
+compute: <2>
+- name: worker
+  replicas: 0 <4>
+metadata:
+  name: nutanix-cluster <5>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <6>
+    hostPrefix: 23 <7>
+  networkType: OVNKubernetes <8>
+  machineNetwork:
+  - cidr: 192.168.111.0/24
+  serviceNetwork: <9>
+  - 172.30.0.0/16
+platform:
+  nutanix:
+    apiVips:
+    - 192.168.111.5
+    ingressVips:
+    - 192.168.111.4
+    prismCentral:
```

Comment:
Should we also explain all the fields under Nutanix, or is that not required for the sample file?